package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "npm run clean && tsc",
     "clean": "npx rimraf dist",
-    "test": "jest --detectOpenHandles --forceExit",
+    "test": "jest",
```

Comment:
omitting the debug options for jest made tests >20% faster on my machine

Reply:
It might even be a good idea to not have them. Since then you could actually notice if one of your tests doesn't exit.